src/ir_Mitsubishi.h

```diff
@@ -87,15 +88,21 @@ union Mitsubishi144Protocol{
     uint8_t WeeklyTimer :1;
     uint8_t             :4;
     // Byte 14
-    uint8_t          :8;
+    uint8_t               :5;
+    uint8_t Ecocool       :1;
+    uint8_t               :2;
     // Byte 15
-    uint8_t          :5;
-    uint8_t iSave10C :1;  //i-SAVE: 1 if in Heat mode AND iSave os on, and temperature THEN set to 10C
-    uint8_t          :2;
+    uint8_t DirectIndirect:2;
+    uint8_t AbsenseDetect :1;
+    uint8_t               :2;
+    uint8_t iSave10C      :1; //i-SAVE:mode=Heat AND iSave=on AND 10C on remote
```

Comment:
Always a space after `//`.
i.e. `// i-SAVE:mode=Heat AND iSave=on AND 10C on remote`